### DISCUSSION TITLE
In 2.6.0 and 2.6.1 route model binding is lost after redirecting back to the previous page.

### DISCUSSION THREAD

**Discussion by DougThwaites** [livewire/livewire#3733] [7. Bugs (archived)] [answered]
After submitting a form request when redirecting back to the form and then loading any Livewire component view data set by route model binding and then a view composer is lost.

In my case I am using route model binding which is then picked up by a view composer and passed to the applicable views.

The view data is a model all the way up until a/any Livewire component is rendered and then it goes back to its string representation in the url.

I have tested this with only a single empty component on the page and still the view data goes from a model back to a string as soon as the livewire component is rendered.

Tested on Livewire 2.6.0 and 2.6.1 reverting back to 2.5.5 corrects this.

I'll work towards a simple code example, hopefully there is something I am still missing and its just my use case.

As always thank you @calebporzio and the team for all of the awesome work :)

**Comment by joshhanley** [comment, marked as answer]
@DougThwaites I suspect that is related to this issue #3725

I have tagged v2.6.2 which fixes that issue 🙂

Please test you issue again and let us know if it's still happening.

Thanks!

**Reply by DougThwaites** [reply]
Correct you are upgraded to 2.6.3 and that solved this particular issue, I noticed something different after upgrading to 2.6.3 but i'll look for an existing discussion or start a different one.

Thanks heaps @joshhanley :)

**Reply by joshhanley** [reply]
@DougThwaites awesome, no worries! Thanks for confirming 🙂
